src/index.json

```diff
@@ -88,9 +88,10 @@
                     "metadata_version": "2.0",
                     "name": "aks-preview",
                     "summary": "Provides a preview for upcoming AKS features",
-                    "version": "0.4.25"
+                    "version": "0.4.26"
                 },
-                "sha256Digest": "58f04bc4417783b102ae83481338e9a16567a9183a021dff5a1f6c444b591f80"
+                "sha256Digest": "572dfc10ff045b00a0e65528afd415b14dfaade3b8d51054b7fc45c13e81f61c"
+                    "version": "0.4.25"
```

Comment:
why there is ` "version": "0.4.25"`? wrong push?

Reply:
oh, wait, seems like we are doing sth at the same time
i was resolving conflicts

Reply:
ok, could you push proper version? i will merge then

Reply:
ok, I will leave this PR to you, thanks.

Reply:
`https://azurecliaks.blob.core.windows.net/azure-cli-extension/aks_preview-0.4.26-py2.py3-none-any.whl` is the correct file now, just remove `"version": "0.4.25"` should be ok